IM_calculation/source_site_dist/src_site_dist.py

```diff
@@ -218,6 +218,66 @@ def calc_rx_ry_GC2(
     return r_x, r_y
 
 
+def calc_rx_ry_GC2_vectorised(
```

Comment:
"vectorised" is too generic as a suffix (The original function was already vectorised over sites :P). Suggest calling it multi_hypocentre or something, to show difference from the original algorithm